### ISSUE TITLE
aboult BSConv-S

### ISSUE THREAD

**Issue by rangmiao** [zeiss-microscopy/BSConv#1]
hello,when i see this paper, i have a problem aboult BSConv-S , in BSConv-S,here is a choice for using a BN and activate layer , So when to use BN and activate layer in BSConv-S ?

**Comment by zeiss-mamthor** [comment]
Thanks for the note. It is true that in the original derivation in the paper, BSConv-S modules do not apply an activation nor a normalization to the pointwise convolution layers. The code now reflects that behavior (at least for the activation) in the current release. However, we observed that for some network architectures (e.g., ResNets/WRNs) an intermediate normalization can be beneficial and help to stabilize training. Such a normalization is in line with the theoretical derivation of the linear subspace transform and we added an option to use batch normalization for intermediate point-wise layers for BSConv-S and BSConv-U.
